kytos/core/api_server.py

```diff
@@ -59,6 +59,24 @@ def register_rest_endpoint(self, url, function, methods):
             url = url[1:]
         self._start_endpoint(f'/kytos/{url}', function, methods=methods)
 
+    def remove_rest_endpoints(self, napp_id):
+        """Remove a rest endpoint from flask.
+
+        This method will remove all endpoints that contain the napp_id.
+
+        Args:
+            napp_id: String using the format '<username>/<napp_name>'
+
+        """
+        for index, rule in enumerate(self.app.url_map.iter_rules()):
+            if napp_id in rule.rule:
```

Comment:
I  think it would be better to check if `rule.rule` "starts" with `/api/<napp_id>` instead of just 'containing' it.

Because if `napp_id = "kytos/test"` and `rule.rule = "/api/diraol/test42/kytos/test"` would create a false positive.

Reply:
@diraol, I think it could be even better if it behaves exactly like the `register_napp_endpoints`, so we are sure that all and only the endpoints started by the decorator will be removed.